src/setup.ts

```diff
@@ -17,13 +17,9 @@ export function setupTest (options: Partial<NuxtTestOptions>) {
   })
 
   afterAll(async () => {
-    if (ctx.nuxt) {
-      await ctx.nuxt.close()
-    }
-
-    if (ctx.browser) {
-      await ctx.browser.close()
-    }
+    await ctx.browser?.close()
+    await ctx.listener?.close()
+    await ctx.nuxt?.close()
```

Comment:
I have doubts in the closing sequence.
I believe the browser should be the last, no?

Reply:
Good question. My thinking was: browser might depend on server and server might depend on something what was created with nuxt instance. In theory last-in-first-out sequence makes sense. I was testing the opposite too, but could not proof any difference. So it could be left how you prefer.